src/app/zap-templates/templates/app/im-cluster-command-handler.zapt

```diff
@@ -37,7 +37,12 @@ Protocols::InteractionModel::Status DispatchServerCommand(CommandHandler * apCom
     {{/first}}
         case Commands::{{asUpperCamelCase commandName}}::Id: {
             Commands::{{asUpperCamelCase commandName}}::DecodableType commandData;
+
+            {{#if isFabricScoped~}}
+            TLVError = DataModel::Decode(aDataTlv, commandData, apCommandObj->GetAccessingFabricIndex());
```

Comment:
I have to ask: do we even need the new DataModel::Decode overload?  Why can't we just `commandData.Decode(...)` here?

Reply:
I tried to minimize how much generated code we have 😅🤣

I.e. if it could be pulled out to a common method instead of being generated for every command, I did so

Reply:
Took a slightly different approach here, adding an overload to Decode method so we don't have to check `isFabricScoped` here; let me know what you think (i.e. if you prefer calling `commandDecode`, will make the change